sql/core/src/test/resources/sql-tests/results/postgreSQL/text.sql.out

```diff
@@ -54,35 +54,37 @@ struct<two:string,f1:string>
 -- !query
 select length(42)
 -- !query schema
-struct<length(42):int>
+struct<>
 -- !query output
-2
+org.apache.spark.sql.AnalysisException
+cannot resolve 'length(42)' due to data type mismatch: argument 1 requires (string or binary) type, however, '42' is of int type.; line 1 pos 7
```

Comment:
hm, `string literal`=>`numeric literals` is allowed, but `numeric literals`=>`string literals` is disallowed?

Reply:
Yes, only string literals are specially handled.